public/v4/apps/freescout.yml

```diff
@@ -0,0 +1,78 @@
+captainVersion: 4
+services:
+    $$cap_appname-db:
+        image: $$cap_db_type:$$cap_database_version
+        volumes:
+            - $$cap_appname-db-data:/var/lib/mysql
+        restart: always
+        environment:
+            MYSQL_ROOT_PASSWORD: $$cap_db_pass
+            MYSQL_DATABASE: freescout
+            MYSQL_USER: $$cap_db_user
+            MYSQL_PASSWORD: $$cap_db_pass
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+
+    $$cap_appname:
+        depends_on:
+            - $$cap_appname-db
+        image: tiredofit/freescout:$$cap_freescout_version
+        volumes:
+            - $$cap_appname-data:/www/html
```

Comment:
Let's remove this as it adds little value and users cannot upgrade simply by deploying new image. This is the directory where the source code lives.